modular_skyrat/modules/antilag/code/kick_subsystem.dm

```diff
@@ -0,0 +1,64 @@
+SUBSYSTEM_DEF(autockick)
```

Comment:
Am I missing something? Why is it `autockick` and not just `autokick`? Seems a bit confusing

Reply:
Hmm, true